editHours.php

```diff
@@ -7,10 +7,13 @@
 $loggedIn = false;
 $accessLevel = 0;
 $userID = null;
+$username = null;
+
 if (isset($_SESSION['_id'])) {
     $loggedIn = true;
     $accessLevel = $_SESSION['access_level'];
     $userID = $_SESSION['_id'];
+    $username = $_SESSION['_id']; // Username is stored here
```

Comment:
userID and username are separate variables but you pull the same data, could you wait to assign username until you need it and replace the instances with userID

Reply:
sounds good!